src/plots/cartesian/constants.js

```diff
@@ -0,0 +1,41 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+module.exports = {
+    // ms between first mousedown and 2nd mouseup to constitute dblclick...
+    // we don't seem to have access to the system setting
+    DBLCLICKDELAY: 600,
+
+    // pixels to move mouse before you stop clamping to starting point
+    MINDRAG: 8,
```

Comment:
Minor point:

I'd vote for making MINDRAG for select box bigger. I find it kind of hard 1d-selecting over large ranges.

Maybe @chriddyp, @cldougl @mdtusz could try it out.